modules/hcp-virt-load-balancer.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy-virt.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-virt-load-balancer_{context}"]
+= Setting up the load balancer
+
+Set up the load balancer service that routes ingress traffic to the KubeVirt VMs and assigns a wildcard DNS entry to the load balancer IP address.
+
+. A `NodePort` service that exposes the hosted cluster ingress already exists. You can export the node ports and create the load balancer service that targets those ports.
+
+.. Get the HTTP node port by entering the following command:
++
+----
+$ oc --kubeconfig <hosted_cluster_name>-kubeconfig get services -n openshift-ingress router-nodeport-default -o jsonpath='{.spec.ports[?(@.name=="http")].nodePort}'
+----
++
+Note the HTTP node port value to use in the next step.
+
+.. Get the HTTPS node port by entering the following command:
++
+----
+$ oc --kubeconfig <hosted_cluster_name>-kubeconfig get services -n openshift-ingress router-nodeport-default -o jsonpath='{.spec.ports[?(@.name=="https")].nodePort}'
+----
++
+Note the HTTPS node port value to use in the next step.
+
+. Create the load balancer service by entering the following command:
++
```

Comment:
Question for technical reviewers: In the following code block, it seems to me that `oc apply -f -` should be separate from the rest of the code. Usually, I've seen that as a command to apply changes to a configuration. Should I reorganize this step into two substeps to say something like this?
```
. Create the load balancer service:
.. Create a YAML file with the following content:
   <YAML content goes here>
.. Apply the changes to the YAML file by entering the following command:
$ oc apply -f file-name.yaml
```